perfkitbenchmarker/providers/gcp/gce_virtual_machine.py

```diff
@@ -283,11 +284,31 @@ def _GenerateCreateCommand(self, ssh_keys_path):
       cmd.flags['machine-type'] = self.machine_type
     cmd.flags['tags'] = 'perfkitbenchmarker'
     cmd.flags['no-restart-on-failure'] = True
-    cmd.flags['metadata-from-file'] = 'sshKeys=%s' % ssh_keys_path
-    metadata = ['owner=%s' % FLAGS.owner]
-    for key, value in self.boot_metadata.iteritems():
-      metadata.append('%s=%s' % (key, value))
-    cmd.flags['metadata'] = ','.join(metadata)
+
+    metadata_from_file = {'sshKeys': ssh_keys_path}
+    for pair in FLAGS.gcp_instance_metadata_from_file:
```

Comment:
Tangent thought, but since we have at least three flags now that will be using this behavior, it may be good to separate it out into a helper. Someone can address that in a separate PR, though.